drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -128,8 +202,20 @@ int spi_mcux_configure(const struct device *dev, const struct spi_config *spi_cf
 	return 0;
 }
 
+static void lpspi_module_system_init(LPSPI_Type *base)
+{
+#ifdef LPSPI_CLOCKS
+	CLOCK_EnableClock(lpspi_get_clock(base));
```

Comment:
Why is this being added. Isn't the clock enabled during bootup in `soc.c` for example?

Reply:
no

Reply:
also ultimately the driver should ungate the clock, not soc.c anyways

Reply:
We should not have a mixed implementation where for some IP blocks the clock enable happens in the driver and for some inside `soc.c`.

Reply:
we already have that, like I said, currently the HAL driver usually ungates the clock on most of the platforms, and soc.c muxes the clock

Reply:
resolving because this commit is already merged by a different PR